### DISCUSSION TITLE
类似tango的低代码平台对于前端工程师的提效之路该如何走？是伪需求吗？

### DISCUSSION THREAD

**Discussion by ZKK666** [NetEase/tango#121] [General]
目前对于tango这种低代码平台对于前端开发工程师的提效是否可观确实存在一些疑虑。简单的页面其实很快可以自己写完，复杂的页面或者个性化的页面，低代码或许不好支持。低代码对于前端工程师的提效这条路不好走。当前如果是非前端人员的话，这种低代码平台提效应该还是很明显的，减少了前端知识的学习成本。

**Comment by debuggerccai** [comment]
或许 低代码+AI 的下一代低代码平台才是未来？另外，围绕低代码平台建立起的全链路生态：域名注册、云服务器托管、Serverless 函数计算、对象存储（OSS）、OCR 识别等付费服务比低代码本身更具备商业价值

**Comment by wwsun** [comment]
或许未来没有所谓的前端工程师
